Controllers/Route.go

```diff
@@ -30,6 +30,7 @@ func (h *Handler) InitRoutes() *gin.Engine {
 	courier := r.Group("/courier")
 	{
 		courier.GET("/", h.GetOneCourier)
+		courier.POST("/create", h.SaveCourier)
```

Comment:
According to REST naming convention you don't need this `create` suffix, POST method already means that you are creating a courier